src/stores/root.ts

```diff
@@ -229,6 +229,13 @@ export class RootStore {
 					spotPriceDestDenom: 'uosmo',
 					destCoinId: 'pool:uosmo',
 				},
+				{
```

Comment:
The price of base coin (comdex) will be shown on the LBP section. And, it should be important to user who is considering buying comdex. So, please check that this is correct.

Reply:
Is correct.